kdesk/icon-hooks.sh

```diff
@@ -64,12 +64,12 @@ case $icon_name in
     fi
 
     # Update the icon with user's avatar and experience level icon
-    if [ "$progress_file" != "" ]; then
-        printf "Icon: $progress_file\n"
+    if [ "$progress_image_path" != "" ]; then
+        printf "Icon: $progress_image_path\n"
     fi
 
-    if [ "$avatar_file" != "" ]; then
-        printf "IconStamp: {13,13} $avatar_file\n"
+    if [ "$avatar_image_path" != "" ]; then
+        printf "IconStamp: {13,13} $avatar_image_path\n"
```

Comment:
I added a fix for the variable names yesterday, so this is working now, but feel free to leave the renaming in